### PR TITLE
Update IDN_ORG_USER_INVITE_ROLE_ASSIGNMENT table

### PR DESCRIPTION
**Pull request by ChanikaRuchini**
### Proposed changes in this pull request

Update IDN_ORG_USER_INVITE_ROLE_ASSIGNMENT table

### Related Issue

- https://github.com/wso2/product-is/issues/18648

### CODE REVIEW

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql [hunk 1]
```diff
@@ -1832,11 +1832,12 @@ CREATE TRIGGER IDN_ORG_USER_INVITATION_TRIG NO CASCADE
...
 /
-CREATE TABLE IDN_ORG_USER_INVITE_ROLE_ASSIGNMENT (
+CREATE TABLE IDN_ORG_USER_INVITE_ASSIGNMENT (
 	ID INTEGER NOT NULL,
 	INVITATION_ID VARCHAR(40) NOT NULL,
-	ROLE_ID VARCHAR(255) NOT NULL,
-	PRIMARY KEY (INVITATION_ID, ROLE_ID),
+    ASSIGNMENT_ID VARCHAR(255) NOT NULL,
+    ASSIGNMENT_TYPE VARCHAR(255) NOT NULL,
+    PRIMARY KEY (INVITATION_ID, ASSIGNMENT_ID),
```
Comment (dewniMW) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql:1840]: alignment issue

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql [hunk 2]
```diff
@@ -1832,11 +1832,12 @@ CREATE TRIGGER IDN_ORG_USER_INVITATION_TRIG NO CASCADE
                 SET (NEW.ID) = (NEXTVAL FOR IDN_ORG_USER_INVITATION_SEQ);
             END
 /
-CREATE TABLE IDN_ORG_USER_INVITE_ROLE_ASSIGNMENT (
+CREATE TABLE IDN_ORG_USER_INVITE_ASSIGNMENT (
 	ID INTEGER NOT NULL,
 	INVITATION_ID VARCHAR(40) NOT NULL,
-	ROLE_ID VARCHAR(255) NOT NULL,
-	PRIMARY KEY (INVITATION_ID, ROLE_ID),
+    ASSIGNMENT_ID VARCHAR(255) NOT NULL,
```
Comment (dewniMW) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql:1838]: several alignment issues in the files

Comment (ChanikaRuchini) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql]: Fixed

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql (hunk 1 above)
Comment (ChanikaRuchini) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql]: Fixed

**Review by dewniMW — APPROVED** [review]

**Pull request merged**